dns_filter.py

```diff
@@ -143,7 +145,9 @@ def main():
     )
     p.add_option('--verbose', '-v', help="log incoming requests to the console", default=False, action="store_true")
     p.add_option('--port', '-p', help="DNS server port number",  default=10053, type='int')
-    p.add_option('--to', '-t', help="destination CNAME", default="127.0.0.1")
+    p.add_option('--cname', '-t', help="destination CNAME", default="127.0.0.1")
```

Comment:
you also need to pass that field instead of `to` further to the `DynamicResolver()`

Reply:
totally right. changed it